testutil/car.go

```diff
@@ -62,6 +67,29 @@ func CreateRandomFile(dir string, rseed, size int) (string, error) {
 	return file.Name(), nil
 }
 
+func CreateRandomUnixfsFileInCar(t *testing.T, dir string, rseed, size int) (string, unixfs.DirEntry) {
```

Comment:
`testing.T` is a little out of place in this file, I hope this is OK. The utilities in github.com/ipfs/go-unixfsnode/testutil need one (although I'm relaxing that for an interface [here](https://github.com/ipfs/go-unixfsnode/pull/53)).

Reply:
This should be okay.